src/dotnet/CarbonAware.DataSources.WattTime/test/WattTimeDataSourceTests.cs

```diff
@@ -65,8 +67,7 @@ public async Task GetCarbonIntensity_ReturnsResultsWhenRecordsFound()
             endDate)
         ).ReturnsAsync(() => emissionData);
 
-        this.LocationConverter.Setup(r => r.ConvertLocationToBalancingAuthorityAsync(location)).ReturnsAsync(balancingAuthority);
-
+        setupBalancingAuthority(balancingAuthority, location);
```

Comment:
non-blocking comment: I really like this new contract of calling the balancing authority converter